core/templates/domain/objects/objects-domain.constants.ts

```diff
@@ -18,24 +18,18 @@
 
 export const ObjectsDomainConstants = {
   FRACTION_PARSING_ERRORS: {
```

Comment:
Probably should rename to something like FRACTION_PARSING_ERROR_I18N_KEYS, ditto below for the other dicts too.

Reply:
Done